CMakeLists.txt

```diff
@@ -1,17 +1,16 @@
-## ---------------------------------------------------------------------
+## ------------------------------------------------------------------------
 ##
+## SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
I haven't seen this before, but assume it's a standardized format. Why does it only list the LGPL if we dual-license things?

Reply:
Yes, this identifier comes from the Linux kernel and is an attempt at unifying license header information. And it is standardized, https://spdx.org/, https://spdx.org/licenses/.